7Cav-Alive.Altis/scripts/missionEnd/missionEnd_StrongPoint.sqf

```diff
@@ -0,0 +1,45 @@
+/* This script will:

```

Comment:
Move this file to `scripts/client/missionEndStrongPoint.sqf`

Be sure to rename the init scripts in `mission.sqm` as well.